### PR TITLE
Add mentor

### PR DESCRIPTION
**Pull request by chimosky**
@JuiP kindly review.

### CODE REVIEW

File: Ideas-2021.md [hunk 1]
```diff
@@ -96,18 +96,13 @@ Suggesting or adding features, fixing bugs, or releasing activities
 will help you to gain experience
 
 **Coding Mentors**<br>
-[Jui Pradhan](https://github.com/JuiP), [Srevin Saju](https://github.com/srevinsaju), [Saumya Mishra](https://github.com/Saumya-Mishra9129)
-
+[Jui Pradhan](https://github.com/JuiP), [Srevin Saju](https://github.com/srevinsaju), [Saumya Mishra](https://github.com/Saumya-Mishra9129), [Ibiam Chihurumnaya](https://github.com/chimosky/)
```
Comment (ACM-BPHC) [inline Ideas-2021.md:99]: Deleting the new line is shifting the Assisting Mentors title to the same line...

**Review by ACM-BPHC — APPROVED** [review]

File: Ideas-2021.md (hunk 1 above)
Comment (JuiP) [inline Ideas-2021.md:99]: Removing the new line shifts assisting mentors to the same line...

**Review by JuiP — APPROVED** [review]

File: Ideas-2021.md (hunk 1 above)
Comment (chimosky) [reply Ideas-2021.md:99]: Didn't notice before, thanks for pointing it out.

Comment (chimosky) [reply Ideas-2021.md:99]: Fixed in 3f565ac.

**Pull request merged**